### ISSUE TITLE
WakaTime CLI download failed

### ISSUE THREAD

**Issue by HMAZonderland** [wakatime/WakaTime.novaextension#15]
Thanks for the new release.

Still seeing this in my console:

```
WakaTime[9:59:17.985000] [WakaTime] [DEBUG] Initializing version 2.1.0

WakaTime[9:59:17.994000] [WakaTime] [DEBUG] Finished initializing WakaTime extension

WakaTime[9:59:18.929000] [WakaTime] [DEBUG] Current wakatime-cli version is v1.38.0

WakaTime[9:59:18.929000] [WakaTime] [DEBUG] Checking for updates to wakatime-cli...

WakaTime[9:59:19.407000] [WakaTime] [DEBUG] GitHub API response 304

WakaTime[9:59:19.408000] [WakaTime] [DEBUG] wakatime-cli is up to date

WakaTime[9:59:19.408000] [WakaTime] [DEBUG] Found new wakatime-cli version: 

WakaTime[9:59:19.485000] [WakaTime] [DEBUG] GitHub API response 304

WakaTime[9:59:19.486000] [WakaTime] [DEBUG] Downloading wakatime-cli from https://github.com/wakatime/wakatime-cli/releases/download/v1.38.0/wakatime-cli-darwin-amd64.zip

WakaTime[9:59:19.487000] [WakaTime] [DEBUG] Finished installing wakatime-cli

WakaTime[9:59:19.487000] [WakaTime] [DEBUG] Downloading wakatime-cli from https://github.com/wakatime/wakatime-cli/releases/download//wakatime-cli-darwin-amd64.zip

WakaTime[9:59:19.906000] [WakaTime] [DEBUG] Extracting wakatime-cli.zip file...

WakaTime[9:59:19.918000] [WakaTime] [ERROR] Failed to extract wakatime-cli.zip with error:   End-of-central-directory signature not found.  Either this file is not

  a zipfile, or it constitutes one disk of a multi-part archive.  In the

  latter case the central directory and zipfile comment will be found on

  the last disk(s) of this archive.

unzip:  cannot find zipfile directory in one of <home>.wakatime/wakatime-cli.zip or

       <home>/.wakatime/wakatime-cli.zip.zip, and cannot find <home>/.wakatime/wakatime-cli.zip.ZIP, period.

WakaTime[9:59:19.921000] [WakaTime] [DEBUG] Finished installing wakatime-cli

WakaTime[9:59:21.767000] [WakaTime] [DEBUG] Extracting wakatime-cli.zip file...

WakaTime[9:59:22.126000] [WakaTime] [DEBUG] Finished installing wakatime-cli

```

After which I do not get the idea the extension is functional. I'm not seeing any pings in my Wakatime dashboard.

**Comment by gandarez** [comment]
Hi, Does your machine is M1 processor?

**Comment by HMAZonderland** [comment]
Yes.

**Comment by gandarez** [comment]
That's the problem, we're in contact with Nova support to make sure we can get the macOS architecture to remove the hard-coded `amd64` from the download function.

**Comment by alanhamlett** [comment]
Fixed in v2.2.0.

**Comment by HMAZonderland** [comment]
All good now! Thanks!

```
[WakaTime] [DEBUG] Sending heartbeat:
<home>/.wakatime/wakatime-cli-darwin-amd64 --entity <some-project-file> --plugin "\"nova/9.1 nova-wakatime/2.1.0"" --language php --local-file /tmp/1649173119892
```
